packages/react/src/ActionMenu/ActionMenu.tsx

```diff
@@ -147,6 +147,17 @@ const Overlay: React.FC<React.PropsWithChildren<MenuOverlayProps>> = ({
   const containerRef = React.useRef<HTMLDivElement>(null)
   useMenuKeyboardNavigation(open, onClose, containerRef, anchorRef)
 
+  // If the menu anchor is an icon button, we need to label the menu by tooltip that also labelled the anchor.
```

Comment:
Handling the case where the anchor is an icon button that has a tooltip by default. We need to make sure menu is labelled by the tooltip since we can't chain aria-label and aria-labelledby ([reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby#:~:text=The%20aria%2Dlabelledby%20property%20cannot%20be%20chained.%20If%20an%20element%20with%20aria%2Dlabelledby%20references%20another%20element%20that%20also%20has%20aria%2Dlabelledby%2C%20the%20aria%2Dlabelledby%20attribute%20on%20the%20referenced%20element%20is%20ignored.))

Let me know if you have any other ideas to resolve this 🙌🏻